object/check.go

```diff
@@ -80,9 +70,7 @@ func CheckUserSignup(application *Application, organization *Organization, authF
 				return i18n.Translate(lang, "check:Email cannot be empty")
 			}
 		} else {
-			if HasUserByField(organization.Name, "email", authForm.Email) {
-				return i18n.Translate(lang, "check:Email already exists")
-			} else if !util.IsEmailValid(authForm.Email) {
+			if !util.IsEmailValid(authForm.Email) {
```

Comment:
we will check whether Email item is visible in signupitem firsrt currently , but in the func `CheckExistingUserDetails` you haven't check it, will it make more problem?